code/modules/mob/living/simple_animal/hostile/mothership_creatures.dm

```diff
@@ -442,13 +486,300 @@
 	else
 		..()
 
+/mob/living/simple_animal/hostile/retaliate/gym_rat/reagent_act(id, method, volume) // Gym rats have to keep away from soymilk... it's bad for their gainz
+	if(isDead())
+		return
+
+	.=..()
+
+	switch(id)
+		if(SOYMILK)
+			if(maxHealth >= 20)
+				visible_message("<span class='warning'>[src] seems to shrink as the soymilk washes over them! Its muscles look less visible...</span>")
+				maxHealth-=5
+			if(maxHealth < 20)
+				visible_message("<span class='warning'>[src] shrinks back into a more appropriate size for a mouse.</span>")
+				transmogrify()
+
 /mob/living/simple_animal/hostile/retaliate/gym_rat/New() // speaks mouse
 	..()
 	languages += all_languages[LANGUAGE_MOUSE]
 
 /mob/living/simple_animal/hostile/retaliate/gym_rat/mothership // Mothership faction version, so it doesn't get attacked by the vault dwellers
 	faction = "mothership"
 
+/mob/living/simple_animal/hostile/retaliate/gym_rat/pompadour // Gym rat with a fancy hairstyle
+	name = "pomdadour rat"
+	desc = "Dang! That's a pretty hunky mouse, let me tell ya."
+	icon_state = "gymrat_pompadour"
+	icon_living = "gymrat_pompadour"
+	icon_dead = "gymrat_pompadour-dead"
+
+	icon_eat = "gymrat_pompadour-eat"
+
+///////////////////////////////////////////////////////////////////ROID RAT///////////
+// That mouse is shredded! Has the science of bodybuilding gone too far? Possibly too swole to control!
+/mob/living/simple_animal/hostile/retaliate/roid_rat
+	name = "roid rat"
+	desc = "It's yoked! Holy shit!"
+	icon_state = "roidrat"
+	icon_living = "roidrat"
+	response_help  = "massages the"
+	response_disarm = "stares jealously at the"
+	response_harm   = "angrily kicks the"
+	treadmill_speed = 3 // CARDIO IS FOR DWEEBS
+	health = 150 // Damn, brother
+	maxHealth = 150
+	speak_chance = 2
+	turns_per_move = 5
+	see_in_dark = 6
+	speak = list("I'M A LEAN, MEAN, WEIGHT PUMPIN'MACHINE.","I BEEN TO THE TOP OF THE MOUNTAIN!","NOTHING MEANS NOTHING!","MAX YOUR PUMP.","OH YEAAAAAH.","CHECK OUT MY PECS, LITTLE MAN.")
```

Comment:
You could make it so the font is larger to account for the screaming.

Reply:
It's alright as is. The additional volume is accounted for by the roid rats squeaking "thunderously" when they say something, instead of squeaking "loudly" like the gym rats.